VAMobile/src/screens/BenefitsScreen/Letters/BenefitSummaryServiceVerification/BenefitSummaryServiceVerification.test.tsx

```diff
@@ -93,128 +86,34 @@ context('BenefitSummaryServiceVerification', () => {
-
-      switchIcon.props.onPress()
-      expect(rnSwitch.props.value).toEqual(true)
-    })
-  })
-
-  describe('on monthly award switch click', () => {
-    it('should update that switches on value', async () => {
-      const switchIcon = testInstance.findAllByType(Switch)[1]
-      const rnSwitch = testInstance.findAllByType(RNSwitch)[1]
-
-      switchIcon.props.onPress()
-      expect(rnSwitch.props.value).toEqual(false)
-
-      switchIcon.props.onPress()
-      expect(rnSwitch.props.value).toEqual(true)
-    })
-  })
-
-  describe('on combined service connected rating switch click', () => {
-    it('should update that switches on value', async () => {
-      const switchIcon = testInstance.findAllByType(Switch)[2]
-      const rnSwitch = testInstance.findAllByType(RNSwitch)[2]
-
-      switchIcon.props.onPress()
-      expect(rnSwitch.props.value).toEqual(false)
-
-      switchIcon.props.onPress()
-      expect(rnSwitch.props.value).toEqual(true)
-    })
-  })
-
-  describe('on permanently disabled switch click', () => {
-    it('should update that switches on value', async () => {
-      const switchIcon = testInstance.findAllByType(Switch)[3]
-      const rnSwitch = testInstance.findAllByType(RNSwitch)[3]
-
-      switchIcon.props.onPress()
-      expect(rnSwitch.props.value).toEqual(false)
-
-      switchIcon.props.onPress()
-      expect(rnSwitch.props.value).toEqual(true)
-    })
-  })
-
-  describe('on one or more disability switch click', () => {
-    it('should update that switches on value', async () => {
-      const switchIcon = testInstance.findAllByType(Switch)[4]
-      const rnSwitch = testInstance.findAllByType(RNSwitch)[4]
-
-      switchIcon.props.onPress()
-      expect(rnSwitch.props.value).toEqual(false)
-
-      switchIcon.props.onPress()
-      expect(rnSwitch.props.value).toEqual(true)
-    })
+    expect(screen.getByText('Army')).toBeTruthy()
+    expect(screen.getByText('Honorable')).toBeTruthy()
+    expect(screen.getByText('January 01, 1990')).toBeTruthy()
+    expect(screen.getByText('October 01, 1993')).toBeTruthy()
+    expect(screen.getByText('Your current monthly payment is $123.00. The effective date of the last change to your current payment was June 06, 2013.')).toBeTruthy()
+    expect(screen.getByText('Your combined service-connected rating is 88%.')).toBeTruthy()
+    expect(screen.getByText("You aren't considered to be totally and permanently disabled solely due to your service-connected disabilities.")).toBeTruthy()
+    expect(screen.getByText("You don't have one or more service-connected disabilities.")).toBeTruthy()
   })
 
   describe('on click of go to ask va', () => {
     it('should launch external link', async () => {
-      await waitFor(() => {
-        testInstance.findAllByType(TouchableWithoutFeedback)[1].props.onPress()
-        expect(mockExternalLinkSpy).toHaveBeenCalledWith('https://ask.va.gov/')
-      })
+      fireEvent.press(screen.getByText('Go to Ask VA'))
```

Comment:
Since this is a link, you can use `getByRole` which is the preferred when applicable
```suggestion
      fireEvent.press(screen.getByRole('link', { name: 'Go to Ask VA' }))
```

Reply:
I think we should fix this with the unit test ticket